### DISCUSSION TITLE
It looks like #config isn't defined for TypeScript?

### DISCUSSION THREAD

**Discussion by jeremeevans** [nuxt/nuxt#18172] [General]
tsconfig.json has: `"extends": "./.nuxt/tsconfig.json"` but I still get this error. It does build and work... so not breaking things, just wondering if I need to update something.

![image](https://user-images.githubusercontent.com/1747006/140578014-53d97d25-3adb-4f47-a9ad-bb47683adaf0.png)

**Comment by aliuq** [comment]
you can use [`useRuntimeConfig`](https://v3.nuxtjs.org/docs/usage/runtime-config#vue-app) on SFC file
```ts
// nuxt.config.ts
import { defineNuxtConfig } from 'nuxt3'
export default defineNuxtConfig({
  privateRuntimeConfig: {
    titlePrivate: 'Title Private',
  },
  publicRuntimeConfig: {
    title: 'Title',
  },
})
```
``` vue
<script setup lang="ts">
const { title, titlePrivate } = useRuntimeConfig() /* Title, Title Private   */
</script>
```

**Comment by danielroe** [comment]
This should be resolved by https://github.com/nuxt/framework/pull/1783.
